xbmc/cores/AudioEngine/Sinks/AESinkXAudio.cpp

```diff
@@ -276,11 +274,9 @@ unsigned int CAESinkXAudio::AddPackets(uint8_t **data, unsigned int frames, unsi
     return frames;
   }
 
-#ifndef _DEBUG
   /* Get clock time for latency checks */
   QueryPerformanceFrequency(&timerFreq);
```

Comment:
per documentation, "The frequency of the performance counter is fixed at system boot and is consistent across all processors. Therefore, the frequency need only be queried upon application initialization, and the result can be cached." Would you add a commit to make it a class member, or would it be a separate PR?